test/Visualization_Tests/src/Table_Visualisation_Integration_Spec.enso

```diff
@@ -0,0 +1,32 @@
+from Standard.Base import all
+
+from Standard.Table import all
+from Standard.Test import all
+import Standard.Visualization.Table.Visualization
+
+type TestData
+    Value ~data
+
+    t self = self.data
+
+    setup = TestData.Value <|
+        vector1 = Data.list '/Users/adriley/Code/enso-org/enso/app/gui/integration-test/project-view/table-vis-json' '{*.json}'
```

Comment:
The same file is used from UI as well as stdlib tests. That's a nice to connect to two sets of tests.

Reply:
Yes just need to work out how I actually relatively path it correctly :)

Reply:
Try `enso_project.root / '..' / '..' / 'app' / 'gui' / 'integration-test' / 'project-view'`